### PR TITLE
Adding Identity fields for node to cloud agent login

### PR DESCRIPTION
**Pull request by apluchik**
Adding new fields to Identity and IdentityProperties structs for LoginConfig creation.

### CODE REVIEW

File: services/security/identity/identity.go [hunk 1]
```diff
@@ -9,15 +9,29 @@ import (
...
 	"github.com/microsoft/moc/pkg/status"
 	wssdcloudsecurity "github.com/microsoft/moc/rpc/cloudagent/security"
+	wssdcloudcommon "github.com/microsoft/moc/rpc/common"
 )
 
 func getIdentity(id *wssdcloudsecurity.Identity) *security.Identity {
+	clitype := security.ExternalClient
+	if id.ClientType == wssdcloudcommon.ClientType_CONTROLPLANE {
+		clitype = security.ControlPlane
+	} else if id.ClientType == wssdcloudcommon.ClientType_EXTERNALCLIENT {
```
Comment (madhanrm) [inline services/security/identity/identity.go:19]: ExternalClient and Node is not same right?

Comment (apluchik) [reply services/security/identity/identity.go:19]: good catch. This should be checking for ClientType_NODE, and external client should be default (external client will have faster expiry/more restrictions in the future).

**Review by madhanrm — APPROVED** [review]

File: services/security/security.go [hunk 2]
```diff
@@ -283,6 +291,14 @@ type CertificateRequest struct {
 type IdentityProperties struct {
 	// State - State
 	Statuses map[string]*string `json:"statuses"`
+	// CloudAgent FQDN
+	CloudFqdn *string `json:"cloudfqdn,omitempty"`
+	// CloudAgent port
+	CloudPort *int32 `json:"cloudport,omitempty"`
+	// CloudAgent authentication port
+	CloudAuthPort *int32 `json:"cloudauthport,omitempty"`
```
Comment (cleonard-git) [inline services/security/security.go:299]: Ports can be uint . Any specific reason for using signed int?

Comment (apluchik) [reply services/security/security.go:299]: Good point. It's added as int32 to stay consistent with existing definitions for Ports our repo, as well as the azure-sdk-for-go repo.

**Pull request merged**